pkg/controller/hybriddeployable/hybriddeployable_template.go

```diff
@@ -259,10 +259,52 @@ func (r *ReconcileHybridDeployable) updateObjectForDeployer(templateobj *unstruc
 
 		obj.SetUnstructuredContent(uc)
```

Comment:
we shall update this deployable section too. it is creating a new deployable instance rather than using existing. https://github.com/IBM/hybriddeployable-operator/blob/76654f72fffa26cee52ce27881fc38b0c6196d27/pkg/controller/hybriddeployable/hybriddeployable_template.go#L242